lib/puppet/face/module/install.rb

```diff
@@ -119,6 +119,10 @@
       EOT
     end
 
+    option '--strict-semver' do
+      summary _('Whether version ranges should exclude pre-release versions')
+    end
```

Comment:
No, the setting is picked up by `Puppet::ModuleTool::Applications::Installer` [here](/puppetlabs/puppet/pull/5883/files#diff-90a94f556f7339081d9d810d564e9800R28) and then used [here](/puppetlabs/puppet/pull/5883/files#diff-90a94f556f7339081d9d810d564e9800R63), [here](/puppetlabs/puppet/pull/5883/files#diff-90a94f556f7339081d9d810d564e9800R107), [here](/puppetlabs/puppet/pull/5883/files#diff-90a94f556f7339081d9d810d564e9800R115), [here](/puppetlabs/puppet/pull/5883/files#diff-90a94f556f7339081d9d810d564e9800R194), and [here](/puppetlabs/puppet/pull/5883/files#diff-90a94f556f7339081d9d810d564e9800R209).